CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.8.0
+
+### Major changes
+
+- New [i18n](https://github.com/bem/bem-core/tree/v2/common.blocks/i18n) block was introduced, providing support for internationalization ([#1074](https://github.com/bem/bem-core/issues/1074)).
```

Comment:
Added very [basic notes](https://github.com/bem/bem-core/blob/v2/common.blocks/i18n/i18n.ru.md) for now. Hope @innabelaya will improve them later on.